messaging/src/main/java/org/frankframework/jms/AbstractJmsListener.java

```diff
@@ -86,6 +88,9 @@ public abstract class AbstractJmsListener extends JMSFacade implements HasSender
 
 	@Override
 	public void configure() throws ConfigurationException {
+		if (getReceiver().isTransacted()) {
+			setTransacted(true);
+		}
```

Comment:
Als we zo een `EnableXA` interface toevoegen (zie comment bij Receiver) dan hoeft dit hier ook niet meer.
Hoewel er wel voor te zeggen is om deze logica in de listener te zetten -- slimmere listeners betekenen minder special cases in de Receiver. Dan zou ik het ook gelijk voor de JdbcListener classes will zien eigenlijk 😬